### PR TITLE
Add onChangeContentSize callback

### PR DESCRIPTION
**Pull request by brentvatne**

### CODE REVIEW

File: ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactContentSizeChangedEvent.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
+package com.facebook.react.views.textinput;
```
Comment (jesseruder) [inline ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactContentSizeChangedEvent.java]: put this line under the copyright comment

File: ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java [hunk 2]
```diff
@@ -390,6 +399,7 @@ public ReactTextInputTextWatcher(
       mPreviousText = null;
     }
 
+
```
Comment (jesseruder) [inline ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java]: remove?

File: ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactContentSizeChangedEvent.java (hunk 1 above)
Comment (brentvatne) [inline ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactContentSizeChangedEvent.java]: k done

**Comment by jesseruder** [comment]
looks good!

**Comment by skevy** [comment]
No longer relevant. We'll have this change in exp-latest after I rebase stuff.

**Pull request closed without merging**